flake.nix

```diff
@@ -895,6 +894,8 @@
                 pkgs.rnix-lsp
                 pkgs-unstable.convco
                 pkgs.nodePackages.bash-language-server
+              ] ++ lib.optionals (!stdenv.isAarch64 || !stdenv.isDarwin) [
+                pkgs.semgrep
```

Comment:
Was this an accidental change by rebasing?

Reply:
I'll remove this by rebasing the branch, I've cherry-picked this one from #1600 because I needed it to use `git commit` and it hasn't landed at the time.